chapters/annotations.tex

```diff
@@ -1571,9 +1571,15 @@ \subsubsection{Conversion rules}\doublelabel{conversion-rules}
 \item
   If OldModifier\_i is cardinality(a)=1 the conversion will only be
   applied for a component comp if there is are any inside connections to
-  comp.a
+  comp.
 \end{itemize}
 
+The converted modifiers and existing modifiers are merged such that existing modifiers takes precedence over the result of convertModifiers.
+A diagnostics is recommended if this merging removes some modifiers unless those modifiers are identical or it is the special case of empty OldModifier list.
```

Comment:
```suggestion
A diagnostics is recommended if this merging removes some modifiers unless those modifiers are identical or it is the special case of an empty OldModifier list.
```

Reply:
Ok. Seems I need to be more careful with the grammatical rules next time.